specification/monitor/resource-manager/Microsoft.Insights/stable/2019-03-01/actionGroups_API.json

```diff
@@ -504,9 +504,9 @@
                     "type": "string",
                     "description": "The email address of this receiver."
                 },
-                "useUnifiedSchema": {
+                "useCommonAlertSchema": {
```

Comment:
Are you guys changing a property name in the existing API version? Did ARM approve this breaking change?

Reply:
Hi, we just commit the new api version on 3/1 and this is not use yet, UX guy just started working on the change.
UX raise the request to change the field name on 3/4 , so I raised this pr.

Reply:
This change should be part of the spec for the new API version then. You cannot introduce breaking changes to a currently stable API version.

Reply:
Hi, Then this apiversion 2019-03-01 will never be used. do youm mean create an 2019-03-06？The change is not in RP Prod yet. no one use it now

Reply:
@jianghaolu  shall we involve the ARM team to review the get some suggestion? This is a urgent fix, we're waiting for approval for RP PROD deployment.